### PR TITLE
chore: added PR template

### PR DESCRIPTION
**Pull request by suevalov**
To add some guidance when opening a PR

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Motivation and Context
+
+<!--
+Why is this change required? What problem does it solve?
+If it fixes an open issue, please link to the issue here.
+-->
+
+## Checklist (check all before merging)
+
+- [ ] Both unit and integration tests are passed
```
Comment (anho) [inline .github/PULL_REQUEST_TEMPLATE.md:25]:
```suggestion
- [ ] Both unit and integration tests are passing
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- [ ] There are not breaking changes
```
Comment (anho) [inline .github/PULL_REQUEST_TEMPLATE.md:26]:
```suggestion
- [ ] There are no breaking changes
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+When adding a new method:
+
+- [ ] The new method is exported both thought default and plain CMA clients
```
Comment (anho) [inline .github/PULL_REQUEST_TEMPLATE.md:30]:
```suggestion
- [ ] The new method is exported through the default and plain CMA client
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+- [ ] All new public types are exported from `./lib/export-types.ts`
+- [ ] Added a unit for the new method
```
Comment (anho) [inline .github/PULL_REQUEST_TEMPLATE.md:32]:
```suggestion
- [ ] Added a unit test for the new method
```

**Review by anho — CHANGES_REQUESTED** [review]

**Review by anho — APPROVED** [review]

**Comment by marcolink** [comment]
Should we maybe add a check for reflecting changes in the docs?

**Review by marcolink — APPROVED** [review]
:shipit:

**Pull request merged**